packages/hub/config/structure.sql

```diff
@@ -590,6 +590,22 @@ CREATE TABLE graphile_worker.migrations (
 
 ALTER TABLE graphile_worker.migrations OWNER TO postgres;
 
+--
+-- Name: merchant_infos; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public.merchant_infos (
+    id uuid NOT NULL,
+    merchant_name text NOT NULL,
+    merchant_slug text NOT NULL,
```

Comment:
In the specification, the name for this field is `unique id`. That sounds a bit confusing to me, because `id` is the actually the unique id. So I took the liberty to use the "slug" nomenclature. Let me know if that's ok!